src/core/RecyclerListView.tsx

```diff
@@ -148,13 +148,6 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
 
     constructor(props: P) {
```

Comment:
can you please also add context as the second option param?

Reply:
done